client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -7,11 +7,17 @@ import com.gu.contentapi.client.thrift.ThriftDeserializer
 import com.gu.contentapi.client.utils.QueryStringParams
 import com.gu.contentatom.thrift.AtomType
 import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration.FiniteDuration
+import java.util.concurrent.TimeUnit
+import scala.util.{Failure, Success}
 
 trait ContentApiClient {
   import Decoder._
   import PaginatedApiResponse._
-  
+
+  /**  **/
+  val scheduledExecutor: ScheduledExecutor = new ScheduledExecutor(1)
```

Comment:
If we keep this, let's make it abstract in the trait and provide a default implementation in the default client.